osmhelper/osm_helper.py

```diff
@@ -69,25 +84,19 @@ def read_stops_from_file():
             return {}
 
     @staticmethod
-    def get_routes(route_type, bbox, refresh=False):
+    def get_routes(bbox, tags, refresh=False):
         if refresh:
             print "Start with fresh routes"
             routes = {}
         else:
             routes = OsmHelper.read_routes_from_file()
 
         # get the master routes first
-        route_masters = OsmHelper.get_route_masters(route_type, bbox, refresh)
+        route_masters = OsmHelper.get_route_masters(bbox, tags, refresh)
 
         api = overpy.Overpass()
-
-        result = api.query("""
-        <query type="relation">
-            <has-kv k="route" v="%s"/>
-            <bbox-query e="%s" n="%s" s="%s" w="%s"/>
-        </query>
-        <print/>
-        """ % (route_type, str(bbox["e"]), str(bbox["n"]), str(bbox["s"]), str(bbox["w"])))
+        query_str = 'relation(%s)%s;(._;);out body;' % (bbox, tags)
```

Comment:
Now I am getting this.
```
Traceback (most recent call last):
  File "./osm2gtfs.py", line 118, in <module>
    main()
  File "./osm2gtfs.py", line 61, in main
    args.refresh_route, data.bbox, data.tags)
  File "/home/user/vcs/osm2gtfs/osmhelper/osm_helper.py", line 310, in refresh_route
    (bbox, tags, '["ref" = "' + route_ref + '"]'))
TypeError: cannot concatenate 'str' and 'int' objects
```

> The tag "route" is not in the query unless it is added to the config file.

Maybe, the old tags should than be added to the config file. I'm not sure if all routes really have the `public_transport:version` tag. Using the old tags for now, allows me to test this later and remove the tags if everything works.

Please let me know when you had a chance to test all code paths that were touched by this PR.